src/main/java/com/jcabi/xml/StrictXML.java

```diff
@@ -192,14 +193,32 @@ private static String join(final Iterable<?> iterable, final String sep) {
     private static Collection<SAXParseException> validate(final XML xml) {
         final Collection<SAXParseException> errors =
             new CopyOnWriteArrayList<SAXParseException>();
+        final int amountOfRetries = 3;
```

Comment:
why not `max`? avoid composite variable names, see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#variable-names